src/browser/services/canny.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+export const cannyOptions = {
+  appID: '601d00fb7a41e3035f75e1e8',
+  position: 'right',
+  align: 'top'
+}
+export const CANNY_FEATURE_REQUEST_URL =
+  'https://neo4j-browser.canny.io/feature-requests'
+
+declare global {
+  interface Window {
```

Comment:
second global window declared in this PR, are they automatically merged? or are they only global for this file

Reply:
Typescript interfaces are merge when declare several yes, we could move this declaration to the custom file, not sure what is more structured